src/main/scala/com/codacy/plugins/api/results/Pattern.scala

```diff
@@ -46,6 +46,16 @@ object Pattern {
                                          languages: Option[Set[Language]] = None)
         extends Specification
 
+    def unapply(s: Pattern.Specification): Option[(Pattern.Id,
+                                                   Result.Level,
+                                                   Category,
+                                                   Option[Subcategory],
+                                                   Option[Set[Parameter.Specification]],
+                                                   Option[Set[Language]])] = s match {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Default case not specified.](https://app.codacy.com/gh/codacy/codacy-plugins-api/pullRequest?prid=4791742)